pytket/extensions/cutensornet/tensor_network_convert.py

```diff
@@ -113,12 +113,22 @@ def _get_gate_tensors(self, adj: bool = False) -> DefaultDict[Any, List[Any]]:
         Returns:
             A map between the gate type and corresponding tensor representation(s).
 
+        Raises:
+            RuntimeError: If ``Box`` objects are present in the circuit.
+
```

Comment:
Ah, just a thought: should this "Raises" also appear in the initialiser of `TensorNetwork`? Or if not a "Raises", a "Note" or something. Not essential, up to you.

Reply:
Yeah, I agree, will add.